lib/system/ansi_c.nim

```diff
@@ -50,6 +50,7 @@ when defined(windows):
     SIGINT* = cint(2)
```

Comment:
> This make sure that core dumps can be generated

in which conditions will a core dump be generated, from a nim user perspective?

Reply:
Most of the time it would be `SIGSEGV`, so basically every time you deref a `nil` or a freed pointer.

Reply:
=> continuing discussion in https://github.com/nim-lang/Nim/pull/16712#issuecomment-762011359